internal/cli/atlas/backup/compliancepolicy/encryptionatrest/encryptionatrest.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encryptionatrest
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
+	"github.com/spf13/cobra"
+)
+
+const (
+	active = "ACTIVE"
+)
+
+type combinedStore interface {
+	store.CompliancePolicyEncryptionAtRestUpdater
+	store.CompliancePolicyDescriber
+}
```

Comment:
multiple things, fiest in go interfaces should be named based on what they do, "combinedStore" doesn't tell me anything

Second this approach of defining the store in this file but combining interfaces of a different package is not a great approach 

For context we define the interfaces on the store for easy of mock reuse but this is not the ideal go approach, ideal on go is to define the interface where is going to be used
see https://github.com/mongodb/mongodb-atlas-cli/blob/4be873f35f05c5c77ec43f35d356a3c752bb6aa5/internal/cli/auth/login.go#L61-L64

Either define the correct interface in the store and use it here or define the correct interface here based on method signature and not by combining both, interface here based on interfaces over there)

Reply:
Moved the interface to the store